RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHGC_cfi.py

```diff
@@ -48,7 +48,15 @@
     algoName = cms.string("PFClusterFromHGCalMultiCluster"),
     thresholdsByDetector = cms.VPSet(
     ),
-    clusterSrc = cms.InputTag("hgcalMultiClusters")
+    
+    clusterSrc = cms.InputTag("ticlMultiClustersFromTrackstersMerge"),
+    
```

Comment:
For a better readability, I would remove the new empty lines added here: L51, L53, L55, L59